### ISSUE TITLE
Add path and compass

### ISSUE THREAD

**Issue by elagin** [Zverik/every_door#560]
For ease of orientation in the area it would be great:
* display the path of movement (At least within a day).
* display the rotation angle of the map (compass).
Perhaps this is easier to implement than #255

**Comment by mnalis** [comment]
Absolutely! Recent path of movent (even just a few minutes of GPS track) since it was implemented in StreetComplete https://github.com/streetcomplete/StreetComplete/issues/2209 was the greatest help in correctly orienteering myself there.

I'd love to see it in Every Door too...

**Comment by Zverik** [comment]
The recent path has been implemented in version 5.1. Idk why f-droid takes so long to publish it, on app store and google play already available.

**Comment by mnalis** [comment]
It has now been published at F-droid (there is a [FAQ](https://f-droid.org/docs/FAQ_-_App_Developers/#ive-published-a-new-release-why-is-it-not-in-the-repository) about why it takes some time), so I've tried it quickly.

I think it would be more useful if dots were replaced by line, because just dots by themselves lose the temporal information that lines contain, i.e. which dot happened before which other dot - which is quite useful for helping with user orienting themselves.

See the picture and explanation I've made on https://github.com/Helium314/SCEE/pull/516#issuecomment-2143587198
